docker-compose.yml

```diff
@@ -35,6 +35,7 @@ x-search-api-env: &search-api-env
   RABBITMQ_PASSWORD: guest
   RACK_ENV: production
   REDIS_URL: redis://redis
+  LOG_TO_STDOUT: "false"
```

Comment:
Can we set this to true? Logging to file isn’t the preferred way for these containers. 

Sorry I missed this on the first round. I’m hoping it’ll be pretty painless to switch this but if it starts a rabbit hole we can keep it as false

Reply:
If we want to **now** log to stdout, I will close this PR because the docker image does that by default now in [here](https://github.com/alphagov/search-api/pull/2371). 

Thanks for your input.